templates/master/00-master/baremetal/files/baremetal-haproxy.yaml

```diff
@@ -130,6 +130,14 @@ contents:
           mountPath: "/host"
         - name: kubeconfigvarlib
           mountPath: "/var/lib/kubelet"
+        livenessProbe:
+          initialDelaySeconds: 10
+          exec:
+            command:
+              - /bin/bash
+              - -c
+              - |
```

Comment:
You used #!/bin/bash everywhere else. Why not here?

Reply:
I think we should run bash command in this way, we had an issue with Keepalived Liveness [1] command, and at the end that was the 'winning' syntax.

[1] https://github.com/openshift/machine-config-operator/blob/master/templates/common/baremetal/files/baremetal-keepalived.yaml#L81-#L85